openstack/nova/templates/etc/_nova-policy.json.tpl

```diff
@@ -176,9 +177,9 @@
     "compute_extension:networks:view": "!",
     "compute_extension:networks_associate": "!",
     "compute_extension:os-tenant-networks": "!",
-    "compute_extension:quotas:show": "rule:context_is_viewer",
-    "compute_extension:quotas:update": "rule:context_is_admin",
-    "compute_extension:quotas:delete": "rule:context_is_admin",
+    "compute_extension:quotas:show": "rule:context_is_viewer or rule:context_is_quota_admin",
+    "compute_extension:quotas:update": "rule:context_is_quota_admin",
+    "compute_extension:quotas:delete": "rule:context_is_quota_admin",
```

Comment:
@fwiesel Does that API still exist in nova queens? I cannot find it there. Currently, this is the `legacy_v2` api. If we roll out queens in the next week, this would essentially do nothing, right?

Reply:
I thought, they are still available, but not as an extension.
It would be quota-sets. Not sure, how it translates to the policy though.